pkg/converter/tool/feature.go

```diff
@@ -28,11 +30,13 @@ const (
 	// stream into nydus blob directly, the tar2rafs eliminates the
 	// need to decompress it to a local directory first, thus greatly
 	// accelerating the pack process.
-	FeatureTar2Rafs Feature = "--type tar-rafs"
+	FeatureTar2Rafs  Feature = "--type tar-rafs"
+	FeatureBatchSize Feature = "--batch-size"
 )
 
 var featureMap = map[Feature]string{
-	FeatureTar2Rafs: "v2.2",
+	FeatureTar2Rafs:  "v2.2",
+	FeatureBatchSize: "v2.2",
```

Comment:
We plan to release the `--batch-size` feature in v2.3, but it seems to cause that can't test the feature in the master branch of image-service repo, for example the latest version `nydus-image` in master branch is `v2.2.0-234-g08c3d0fa`.

Reply:
Changed the way of detecting features.